apps/studio/src/features/users/components/UserTableTabs/UserTableTab.tsx

```diff
@@ -0,0 +1,34 @@
+import { Badge, Tab, Text } from "@chakra-ui/react"
+
+interface UserTableTabProps {
+  label: string
+  count: number
+}
+
+export const UserTableTab = ({ label, count }: UserTableTabProps) => {
```

Comment:
[issue] i think this is the wrong way of doing things - refer to chakra's docs [here](https://v2.chakra-ui.com/docs/components/tabs/usage#creating-custom-tab-components) for custom tab components!

at present, this is just causing a work-around for the badge's selected state so not very important but if it's not super difficult, could i trouble you to shift it to use `forwardRef`?

Reply:
thanks for pointing this out, didn't know about this. updated in https://github.com/opengovsg/isomer/pull/1136/commits/e9b9e436274fc6988913afffb00c03a4de1e8ad5